packages/devp2p/src/rlpx/rlpx.ts

```diff
@@ -244,14 +246,16 @@ export class RLPx extends EventEmitter {
 
       if (!disconnectWe && reason === DISCONNECT_REASONS.TOO_MANY_PEERS) {
         // hack
```

Comment:
i wonder why this says "hack"?

Reply:
Me too!  Wasn't my comment

Reply:
If I recall correctly this was me adding this some time ago, I guess this was from a time where client/devp2p connections were still more tricky then today (where we fixed several bugs along the way making devp2p more stable) and I think I added this for situations where there were just no other peers available any more and so I readded existing but disconnecting peers to the queue to at least have a chance to at some point get a connection (which felt a bit hacky to me, therefore the comment).